### ISSUE TITLE
Can you explain about what are the use cases with these scripts?

### ISSUE THREAD

**Issue by ghost** [Mr-Un1k0d3r/ATP-PowerShell-Scripts#1]
Hi,
there isn't a discussion section so I'm opening an issue (even though there is no issue) :)

could you explain a bit about what to do with these scripts and what are they used for? thanks in advance

**Comment by ayiz09** [comment]
Yeah, I want to know about this too. I see one of the ATP scripts was detected as process injection in my EDR.
